src/dsql/parse.y

```diff
@@ -5272,18 +5283,35 @@ set_decfloat_traps
 			{ $$ = $5; }
 	;
 
-%type <setHighPrecBindNode> set_decfloat_bind
-set_decfloat_bind
-	: SET bind_to_type BIND
-			{ $$ = newNode<SetHighPrecBindNode>($2); }
-		decfloat_bind_clause($4)
-			{ $$ = $4; }
+%type <setBindNode> set_bind
+set_bind
+	: SET BIND OF set_bind_from
+			{ $$ = newNode<SetBindNode>(); $$->from = $4; }
```

Comment:
@AlexPeshkoff why do you need to create intermediary action rule here, if the intermediary rule is not passed as parameter later?